### DISCUSSION TITLE
Re download AAX files - ignores existing files.

### DISCUSSION THREAD

**Discussion by acr1161** [openaudible/openaudible#1103] [Q&A] [answered]
I use an external disk to store everything, since I have some 2900 books.  Just setup a new laptop...  Changed the settings to point to the external drive, but now it wants to re-download about 1700 books.  They are already downloaded and are in the AAX folder.  The difference appears to be the names have a dash in them.  So where I have 2287 AD After Destruction Book 1. the books name from audible has a dash...  2287 AD - After Destruction Book 1.  The file downloads as 2287 AD.AAX  It appears that everything from the Dash on is ignored, to the system thinks the existing AAX file is a different book.  I would like to not have to re-download the books.  any thoughts?

**Comment by openaudible** [comment, marked as answer]
Sorry for the delay. We don't get notified on new posts here.
The latest version(s) should fix this. You may need to disable automatic downloads/conversions and use the "Manage Libraries" under the File menu to open the correct library.
